modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TxAbstractTest.java

```diff
@@ -330,7 +330,12 @@ protected TxManager txManager(TableViewInternal t) {
     protected boolean assertPartitionsSame(TableViewInternal table, int partId) {
         long storageIdx = 0;
 
-        for (Map.Entry<String, Loza> entry : txTestCluster.raftServers().entrySet()) {
+        Map<String, Loza> raftServers = txTestCluster.raftServers();
+        if (raftServers == null) {
```

Comment:
I wonder how you stumbled upon this)

Reply:
"localhost" was being resolved to more than 1 address, so more than one set of managers per node was started, so start was failing due to a busy port. So the start method (which is a `@BeforeEach`) was not able to create the `raftService` map. `@AfterEach` method was run then, which is assuming that the start method finished successfully, so an NPE was thrown masking the original problem.